tests/integration/components/links-with-follower-test.js

```diff
@@ -81,12 +86,75 @@ describeComponent(
     });
 
     describe('clicking a link', function() {
-      it.skip('moves follower to the active link position')
-      it.skip('changes width of follower to width of active link');
+      let router;
+
+      beforeEach(function() {
+        this.register('router:main', Ember.Object.extend(Ember.Evented));
+
+        router = this.container.lookup('router:main');
+
+        this.render(hbs`
+          {{#links-with-follower class="icon-nav"}}
+            <li class="active">One</li>
+            <li>Two</li>
+            <li class="long">Three Four Five Six</li>
+          {{/links-with-follower}}
+        `);
+      });
+
+      it('moves follower to the active link position', function(done) {
+        let $follower = this.$('.link-follower');
+
+        run.later(function() {
+          expect($follower.position().left).to.equal(this.$('.active').position().left);
+        }, 10);
+
+        run.later(function() {
+          this.$('.active').removeClass('active');
+          this.$('.long').addClass('active');
+          router.trigger('willTransition');
+        }, 20);
+
+        run.later(function() {
+          expect($follower.position().left).to.equal(this.$('.active').position().left);
+          done();
+        }, 30);
```

Comment:
I don't really like having these `run.later` hooks here - but they need to exist because of `afterRender`. @cball Would you happen to have a recommendation on how to handle these?